test/unit/error.c

```diff
@@ -655,6 +656,99 @@ test_client_error_name(void)
 	footer();
 }
 
+static void
+test_logging(void)
+{
+	header();
+	plan(6);
+
+	char path[] = "/tmp/XXXXXX";
+	mktemp(path);
```

Comment:
You should delete the file when the test ends. Also, probably better name the file so that it can be identified.